apps/web/pages/api/get-customer.ts

```diff
@@ -0,0 +1,165 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import prisma from "@calcom/prisma";
+
+import usageCard from "./plain/customer-card-display";
+
+type Customer = {
+  id: string;
+  email: string;
+  externalId?: string;
+};
+
+type Thread = {
+  id: string;
+  externalId?: string;
+};
+
+type CardRequest = {
+  cardKeys: string[];
+  customer: Customer;
+  thread: Thread;
+};
+
+type CardComponent = {
+  componentText?: {
+    text: string;
+  };
+  componentDivider?: {
+    dividerSpacing?: "relaxed" | "default" | "compact";
+  };
+  componentHeader?: {
+    text: string;
+  };
+};
+
+type CardResponse = {
+  cards: {
+    key: string;
+    timeToLiveSeconds: number;
+    components: CardComponent[];
+  }[];
+};
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  if (req.method !== "POST") {
+    res.setHeader("Allow", ["POST"]);
+    return res.status(405).end(`Method ${req.method} Not Allowed`);
+  }
+
+  try {
+    console.log("Request received:", {
```

Comment:
We should probably get rid of all console.logs that don't log an actual error or important action